TrackingDevice/data/tables/trackingdevice-sct.tbm

```diff
@@ -0,0 +1,185 @@
+		local trackee = hv.Self
+		if trackee:getBreedName() == "Ship" then
+			local tracker_class = td.WeaponToShip[wep.Class.Name]
+			if tracker_class then
+				-- spawn the tracking device and record it
+				td.TotalTracked = td.TotalTracked + 1
+				local tracker_name = tracker_class.Name .. " " .. td.TotalTracked
+				local tracker = mn.createShip(tracker_name, tracker_class, orient, pos, wep.Team)
+				td.TrackerToTrackee[tracker_name] = trackee.Name
+				td.EveryTrackee[trackee.Name] = true
+
+				-- get the place where the weapon hit
+				local ci = wep:getCollisionInformation()
+				local local_pos = ci:getCollisionPoint(true)
+				local submodel = ci:getCollisionSubmodel()
+
+				-- see if we are going to follow a moving submodel
+				local smi
+				if submodel then
+					local mi = trackee.ModelInstance
+					if mi then
+						smi = mi.SubmodelInstances[submodel]
+					end
+				end
+
+				-- "stick" the tracking device on the ship
+				local f
+				if smi then
+					local local_normal = ci:getCollisionNormal(true)
+					local local_orient = ba.createOrientationFromVectors(local_normal)
+					f = function()
+						tracker.Position, tracker.Orientation = smi:findWorldPointAndOrientation(local_pos, local_orient)
+					end
+				else
+					local world_normal = ci:getCollisionNormal()
+					local world_orient = ba.createOrientationFromVectors(world_normal)
+					local offset_orient = trackee.Orientation:getTranspose() * world_orient
+					f = function()
+						tracker.Position = trackee.Position + trackee.Orientation:unrotateVector(local_pos)
+						tracker.Orientation = trackee.Orientation * offset_orient
+					end
+				end
+
+				-- keep the tracker stuck on
+				trackee:addPostMoveHook(f)
+			end
+		end
+	end
+]
+
+$On Ship Depart:
+[
+	-- if there is a tracking device on the departed ship, make it depart too
+	local td = TrackingDevice
+	if td.Enabled then
+		local ship_name = hv.Ship.Name
+		local method = hv.Method
+		local t = td.TrackerToTrackee
+		if t then
+			for tracker_name, trackee_name in pairs(t) do
+				if trackee_name == ship_name then
+					if method == "SHIP_VANISHED" then
+						mn.evaluateSEXP("( ship-vanish !" .. tracker_name .. "! )")
+					else
+						mn.evaluateSEXP("( set-departure-info !" .. tracker_name .. "! !Hyperspace! !<no anchor>! !0! !0! (false) )")
+						mn.Ships[tracker_name]:warpOut()
+					end
+				end
+			end
+		end
+	end
+]
+
+$On Ship Death:
+[
+	-- if either the tracking device or the ship it tracks has been destroyed, remove the entry
+	local td = TrackingDevice
+	if td.Enabled then
+		local ship_name = hv.Ship.Name
+		local t = td.TrackerToTrackee
+		if t then
+			for tracker_name, trackee_name in pairs(t) do
+				-- remove entry
+				if tracker_name == ship_name or trackee_name == ship_name then
+					t[tracker_name] = nil
+				end
+				-- if the trackee was destroyed, destroy any trackers
+				if trackee_name == ship_name then
+					mn.evaluateSEXP("( self-destruct !" .. tracker_name .. "!)")
```

Comment:
as with the other two, but with ship:kill()

Reply:
as above

Reply:
Bleh, and this will require PR 5251.